### DISCUSSION TITLE
Where can i find best ideas about projects for beginners to practice html and css?

### DISCUSSION THREAD

**Discussion by felixcamilo** [community/community#109747] [New to GitHub] [answered]
I'm a beginner in the world of software development and I need a lot of ideas of projects to practice my skills. Can you provide some resources?

**Comment by DeandrePetersen** [comment, marked as answer]
Here is a list of well-known platforms that may help you:
GitHub Repositories: Explore GitHub repositories for beginner-friendly projects and contribute to open-source projects. You can find projects in various programming languages and domains.

Codecademy: Codecademy offers interactive coding lessons and projects in different programming languages. They have guided projects to help you apply what you've learned.

FreeCodeCamp: FreeCodeCamp offers coding challenges, tutorials, and projects in web development, including front-end, back-end, and full-stack projects.

Project Euler: Project Euler provides a series of challenging mathematical/computer programming problems that will require more than just programming skills to solve. It's a great way to enhance your problem-solving abilities.

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @alejandrocamilo  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
